prov/gni/test/rdm_dgram_rma.c

```diff
@@ -731,6 +767,90 @@ Test(dgram_rma, writemsg_retrans)
 	xfer_for_each_size(do_writemsg, 8, BUF_SZ);
 }
 
+void do_writemsg_more(int len)
+{
+	int ret;
+	ssize_t sz;
+	struct fi_cq_tagged_entry cqe = { (void *) -1, UINT_MAX, UINT_MAX,
+					  (void *) -1, UINT_MAX, UINT_MAX };
+	struct iovec iov, iov2;
+	struct fi_msg_rma msg, msg2;
+	struct fi_rma_iov rma_iov, rma_iov2;
+	uint64_t w[2] = {0}, r[2] = {0}, w_e[2] = {0}, r_e[2] = {0};
+
+	iov.iov_base = source;
+	iov.iov_len = len;
+
+	iov2.iov_base = source2;
+	iov2.iov_len = len;
+
+	rma_iov.addr = (uint64_t)target;
+	rma_iov.len = len;
+	rma_iov.key = mr_key[1];
+
+	rma_iov2.addr = (uint64_t)target2;
+	rma_iov2.len = len;
+	rma_iov2.key = mr_key2[1];  /* use different mr_key? */
```

Comment:
Sorry, this isn't my latest code. It has since changed - i'll push it up in a moment.